src/prices/prices-util.ts

```diff
@@ -176,8 +176,16 @@ export const getVaultTokenPrice = async (contract: string): Promise<TokenPrice>
   vaultTokenPrice.name = token.name;
   vaultTokenPrice.address = token.address;
   const sett = vaultTokenSnapshot.sett;
-  vaultTokenPrice.usd *= sett.pricePerFullShare / 1e18;
-  vaultTokenPrice.eth *= sett.pricePerFullShare / 1e18;
+
+  let multiplier;
+  if (token.address === TOKENS.BDIGG) {
```

Comment:
we will probably figure out a better way to encapsulate this stuff later this was needed to some ui integration side of things